examples/Makefile

```diff
@@ -15,4 +15,4 @@ html:   $(HTML)
 	$(OPEN) $@
 
 %.txt:	%.xml
-	xml2rfc  $< $@
+	xml2rfc --text  $<
```

Comment:
Hmm, `xml2rfc --help` says:

```
  One or more of the following output formats may be specified. The default
  is --text. The destination filename will be based on the input filename,
  unless --out=FILE or --basename=BASE is used.

  --text                outputs formatted text to file, with proper page
                        breaks
```

So why is that change desirable?

Reply:
I get an error without it.

Reply:
Oh.  The $@.  But there is no need for the --text.

Reply:
Feel free to remove.